lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -6834,11 +6834,19 @@ namespace Js
                     - If the size is >= 100, scale by 1.6
             */
             const uint loopPercentage = GetByteCodeInLoopCount() * 100 / max(1u, GetByteCodeCount());
-            if(loopPercentage <= 50)
+            const int byteCodeSizeThresholdForInlineCandidate = CONFIG_FLAG(InlineThreshold) + CONFIG_FLAG(InlineThresholdAdjustCountInSmallFunction);
```

Comment:
You should use CONFIG_FLAG(LoopInlineThreshold) here because, today, that is the largest bytecode size for which we will always inline.